project/config.py

```diff
@@ -15,13 +15,41 @@
 
 from functools import lru_cache
 from typing import Any
+from typing import Dict
+from typing import Optional
 
+from common import VaultClient
 from pydantic import BaseSettings
 from pydantic import Extra
 from pydantic import Field
 from pydantic import HttpUrl
 
 
+class VaultConfig(BaseSettings):
+    """Store vault related configuration."""
+
+    APP_NAME: str = 'pipelines'
+    CONFIG_CENTER_ENABLED: bool = False
+
+    VAULT_URL: Optional[str]
+    VAULT_CRT: Optional[str]
+    VAULT_TOKEN: Optional[str]
```

Comment:
Please add default values here.
```suggestion
    VAULT_URL: Optional[str] = None
    VAULT_CRT: Optional[str] = None
    VAULT_TOKEN: Optional[str] = None
```